kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -284,6 +299,19 @@ const List: React.FC<TopicsListProps> = ({
                 >
                   Delete selected topics
                 </Button>
+                {tableState.selectedCount === 1 ? (
+                  <Button
+                    buttonSize="M"
+                    buttonType="secondary"
+                    isLink
+                    to={clusterTopicCopyPath(
+                      `${clusterName}/topics/copy/?${getSelectedTopic()}`
+                    )}
+                  >
+                    Copy selected topic
+                  </Button>
+                ) : null}
```

Comment:
Can we use `tableState.selectedCount === 1 && (...` instead of using ternary and returning null?